conf/ocsci/rosa.yaml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
What about conf/deployment/rosa/ ? Don't we want to have all rosa configs in one place?

Reply:
According to https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/README.md, /conf/ocsci/ dir seems like a place to store common platform configuration.

Reply:
As discussed, if we refer to the code, configurations files of other platforms say AWS platform at `conf/deployment/AWS` which covers the different configurations, might be we can have more rosa specific configuration files at `conf/deployment/rosa/*.yaml` w.r.t AZ, count of masters and worker instances, 1Tib/4Tib cluster size. 
Also in a contradiction to my comment above, for now, we can merge this as a part of the current managed service automation phase. And Maybe improve on it further as separate PRs in the future without disturbing the focus from other priorities automation tasks defined for the current phase. 
For now, I agree to this common conf file at a common platform configuration place.

Reply:
Also, I didn't find the specif documentation stating this use case. So might be we can improve this by adding `https://github.com/red-hat-storage/ocs-ci/tree/master/conf/deployment/README.md` for stating it.